libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -4,7 +4,7 @@
   [formGroup]="formGroup"
 >
   <div class="sky-date-range-picker-form-group">
-    <sky-input-box [disabled]="disabled">
+    <sky-input-box stacked="true" [disabled]="disabled">
```

Comment:
so I think the issue here is that it shouldn't always be stacked. When on small screens it stacks so all but the last input should have stacked=true. When on medium screens it is usually displayed in one row so we wouldn't set it because we don't know if there are more inputs below these.

The branch linked in that other datetime spacing story contains some logic around how to apply spacing here

Reply:
for this PR: keep `stacked="true"`, this issue will considered and fixed with this bug: https://blackbaud.visualstudio.com/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2552087